.circleci/config.yml

```diff
@@ -23,16 +23,29 @@ references:
       at: *workspace_root
 
 commands:
-  terraform-init-then-apply:
-    description: "Initializes and applies terraform configuration"
+  assume-role-and-persist-workspace:
+    description: "Assumes deployment role and persists credentials across jobs"
     parameters:
       aws-account:
         type: string
-      aws-role-name:
-        type: string
+    steps:
+      - checkout
+      - aws_assume_role/assume_role:
+          account: <<parameters.aws-account>>
+          profile_name: default
+          role: "LBH_Circle_CI_Deployment_Role"
+      - persist_to_workspace:
+          root: *workspace_root
+          paths:
+            - .aws
+  terraform-init-then-apply:
+    description: "Initializes and applies terraform configuration"
+    parameters:
       environment:
         type: string
     steps:
+      - *attach_workspace
```

Comment:
Had to look up why the asterisk here.....Monday morning and I've already learned something - very useful!